docs/pipelines/overview/index.md

```diff
@@ -1,15 +1,25 @@
 # What is Gruntwork Pipelines?
 
+Gruntwork Pipelines is a framework for securely deploying infrastructure as code to your AWS environments. Gruntwork Pipelines runs as a binary and series of steps in your CI system, determining what _actions_ need to be taken, in which _environments_, based on the _infrastructure changes_ that occurred.
+
+## Execution vs configuration
```

Comment:
I like this section overall. I'm not sure "execution vs. _configuration_" is quite the right split, though. Also, "vs" is doing a lot of work here, so maybe we should say something more explicit like "Isolating IaC definitions and deployment."

Also, it's implied here, but it's worth stating very explicitly that this is done to provide a strong security model. We should probably even link to the page about access control.

Reply:
I like that title, I was struggling to come up with something reasonable so I just went with the first thing that came to mind. I'll add a some info regarding the security model.

Reply:
Added in [b6f7a1e](https://github.com/gruntwork-io/docs/pull/1099/commits/b6f7a1e0be0b538a04143bb9448cc9012cb751a3)